### PR TITLE
commit for pull request

### PR DESCRIPTION
**Pull request by JavaPsyh**

### CODE REVIEW

File: BooleanMethod.java [hunk 1]
```diff
@@ -3,16 +3,11 @@
 public class BooleanMethod {
```
Comment (Sofasmile) [inline BooleanMethod.java]: doesn't add 'Method' in the class name or method name. Please, change it

File: BooleanMethod.java [hunk 2]
```diff
@@ -3,16 +3,11 @@
...
 
     public static boolean booleanMethod (boolean a, boolean b, boolean c, boolean d){
```
Comment (Sofasmile) [inline BooleanMethod.java]: the same

File: CheckResult.java [hunk 3]
```diff
@@ -1,7 +1,7 @@
 package psyh;
 
-import java.math.BigDecimal;
-import java.math.RoundingMode;
+//import java.math.BigDecimal;
```
Comment (Sofasmile) [inline CheckResult.java]: remove this lines

File: CheckResult.java [hunk 4]
```diff
@@ -1,7 +1,7 @@
...
+//import java.math.RoundingMode;
 
 /**
  * Реализуйте метод, возвращающий ответ на вопрос: правда ли, что a + b = c?
```
Comment (Sofasmile) [inline CheckResult.java]: move 0.0001 into class and make it static final variable.

File: FlipBit.java [hunk 5]
```diff
@@ -2,6 +2,6 @@
 
 public class FlipBit {
     public static int flipBit(int value, int bitIndex) {
-        return value + (int) Math.pow(2, bitIndex-1);
+        return value ^ (1 << bitIndex-1);
```
Comment (Sofasmile) [inline FlipBit.java]: use combination Ctrl+Alt+L to automatically formatting your code

File: LeapYears.java [hunk 6]
```diff
@@ -2,6 +2,6 @@
 
 public class LeapYears {
     public static int quantityOfLeapYears(int year){
-        return year / 4 - year / 100 + year / 400;
+        return year/4 - year/100 + year/400;
```
Comment (Sofasmile) [inline LeapYears.java]: use combination Ctrl+Alt+L to automatically formatting your code

File: BooleanMethod.java (hunk 2 above)
Comment (Sofasmile) [reply BooleanMethod.java]: make another implementation
with boolean operators

File: CheckResult.java [hunk 7]
```diff
@@ -1,7 +1,7 @@
 package psyh;
```
Comment (Sofasmile) [inline CheckResult.java:1]: change the name of the package

**Review by Sofasmile — COMMENTED** [review]
resolve comments
remove all comments in your code which is unnecessary

File: CheckResult.java (hunk 7 above)
Comment (JavaPsyh) [reply CheckResult.java:1]: I won't rename "psyh". Let's close this question.

**Review by Sofasmile — APPROVED** [review]
good

**Pull request merged**